src/card.ts

```diff
@@ -0,0 +1,112 @@
+'use strict'
+
+import type {
+  Marqeta,
+  MarqetaOptions,
+} from './'
+
+export interface ExpirationOffset {
+  unit?: string;
+  value?: bigint;
+}
+
+export interface AddressType {
+  firstName?: string;
+  middleName?: string;
+  lastName?: string;
+  address1: string;
+  address2?: string;
+  city?: string;
+  state?: string;
+  zip?: string;
+  country?: string;
+  phone?: string;
+  postalCode?: string;
+}
+
+export interface Shipping {
+  method?: string;
+  returnAddress?: AddressType;
+  recipientAddress?: AddressType;
+  carOfLine?: string;
+}
+
+export interface NameLineType {
+  [value: string]: string;
+}
+
+export interface Text {
+  nameLine1: NameLineType;
+  nameLine2?: NameLineType
+  nameLine3?: NameLineType
+}
+
+export interface Card {
+  name?: string
+  thermalColor?: string;
+}
+
+export interface ImagesCarrier {
+  name?: string
+  message1?: string;
+}
+
+export interface NameType {
+  name?: string
+}
+
+export interface Images {
+  card?: Card;
+  carrier?: ImagesCarrier;
+  signature?: NameType;
+  carrierReturnWindow?: NameType;
+}
+
+export interface Carrier {
+  templateId?: string;
+  logoFile?: string;
+  logoThumbnailFile?: string;
+  messageFile?: string;
+  messageLine?: string;
+}
+
+export interface CardPersonalization {
+  text: Text;
+  images?: Images;
+  carrier?: Carrier;
+  persoType?: string;
+}
+
+export interface Fulfillment {
+  shipping?: Shipping;
+  cardPersonalization: CardPersonalization;
+}
+
+export interface ActivationActions {
+  terminateReissuedSourceCard?: boolean;
+  swapDigitalWalletTokensFromCardToken?: string;
+}
+
+export interface CardType {
```

Comment:
I think you call this `CardType` as if to say "This is the Card Type"... but it's an interface... so wouldn't this just be the `Card`?